src/internal/modules/billing/controllers/bill-run.js

```diff
@@ -111,11 +116,20 @@ const getBillingBatchCancel = async (request, h) => billingBatchAction(request,
 
 const postBillingBatchCancel = async (request, h) => {
   const { batchId } = request.params;
+
+  try {
```

Comment:
This is not needed here. if the whole batch is cancelled the original invoice remains as marked for re-billing